eng/Versions.props

```diff
@@ -25,24 +25,24 @@
     <MicrosoftVisualStudioExtensibilityTestingVersion>0.1.149-beta</MicrosoftVisualStudioExtensibilityTestingVersion>
     <!-- CodeStyleAnalyzerVersion should we updated together with version of dotnet-format in dotnet-tools.json -->
     <CodeStyleAnalyzerVersion>4.3.0-1.final</CodeStyleAnalyzerVersion>
-    <VisualStudioEditorPackagesVersion>17.4.203-preview</VisualStudioEditorPackagesVersion>
+    <VisualStudioEditorPackagesVersion>17.5.42-preview</VisualStudioEditorPackagesVersion>
```

Comment:
17.5 previews haven't shipped yet so this likely does need to go in some other branch than main unless you can debug and run tests in the latest public preview, can you confirm that works/doesn't work?

Reply:
alternatively we can wait until P1 ships, and fix this in P2 without another branch

Reply:
> alternatively we can wait until P1 ships, and fix this in P2 without another branch

is this alternative to merging into main-vs-deps?